### PR TITLE
Ember model rework

### PR DESCRIPTION
**Pull request by C0rydoras**

### CODE REVIEW

File: ember/app/components/project-detailed/template.hbs [hunk 1]
```diff
@@ -12,6 +12,6 @@
 
   </div>
 
-  <p class='seperator'></p>
-  <DependencyTable @dependencyVersions={{@project.dependencyVersions}} />
+  <p class='seperator' />
```
[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/components/project-detailed/template.hbs]: It is responsive, and also different to UIkits `<hr />` styling in other ways as for Instance the margins.

[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/components/project-detailed/template.hbs]: I changed it from a `<p />` to a `<hr />` here: https://github.com/adfinis/Outdated/pull/24/commits/a22a7fab282649c0aa9ce312a1e259781065bc80

File: ember/app/models/version.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class VersionModel extends Model {
+  @attr patchVersion;
+  @attr('django-date') releaseDate;
+  @belongsTo('releaseVersion') releaseVersion;
+
+  get name() {
+    return this.releaseVersion.get('dependency').get('name');
```
[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/models/version.js]: done here https://github.com/adfinis/Outdated/pull/24/commits/6f6857d73c621c236b68e7527a35f393388a9f3f

File: ember/app/models/version.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  }
+  get endOfLife() {
+    return this.releaseVersion.get('endOfLife');
+  }
+
+  get version() {
+    const releaseVersion = this.releaseVersion;
+    return `${releaseVersion.get('releaseVersion')}.${this.patchVersion}`;
```
[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/models/version.js]: done here https://github.com/adfinis/Outdated/pull/24/commits/6f6857d73c621c236b68e7527a35f393388a9f3f

File: ember/app/styles/components/_text.scss [hunk 4]
```diff
@@ -8,8 +8,11 @@
...
 }
 .text-UNDEFINED {
-  @extend .uk-text-muted;
-  font-style: italic;
+  &:not(tr),
+  td[data-test-version-end-of-life] {
+    @extend .uk-text-muted;
+    font-style: italic;
+  }
 }
```
[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/styles/components/_text.scss:16]: I currently use the status property to set the class ("text-{version.status}")

[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/styles/components/_text.scss:16]: For me it seemed like I would need a lot more code if I went with the built-in classes, but maybe you know a better approach where we could use the built-in classes.

[the remark this answers is not in the document]
Comment (C0rydoras) [reply ember/app/styles/components/_text.scss:7]: Done here https://github.com/adfinis/Outdated/pull/24/commits/ea45e62bc1abb8e12eff1b0f625fbf9a2968c226

**Review by anehx — APPROVED** [review]
LGTM! Make sure to squash and merge.

**Pull request merged**